.circleci/config.yml

```diff
@@ -28,9 +28,10 @@ jobs:
     working_directory: ~/marquez_python
     steps:
       - checkout
+      - run: ./docker/release_check.sh
       - run: pip install --user twine
       - run: python setup.py sdist
-      - run: python -m twine upload --verbose dist/*
+      - run: echo "re-enable this command - python -m twine upload --verbose dist/*"
```

Comment:
TODO: will flip this back to the actual command once the rest of the PR is +1'd.
`- run: python -m twine upload --verbose dist/*`